### DISCUSSION TITLE
虛擬歌手

### DISCUSSION THREAD

**Discussion by ying0607** [1101-GS3073-C/Python-Community-GS3073-C#653] [General]
之前在看YouTube影片的時候發現有些歌手的演出會和虛擬歌手合作
而那些虛擬歌手幾乎可以以假亂真
我發現我有時候甚至會懷疑那個歌手到底是不是真人
看到這篇文章才知道連K-pop都可以有一個全部團員都是虛擬的團體
我原本以為人跟人之間的互動是AI沒辦法取代的
但以這個趨勢看來人類被AI取代感覺只是時間問題了
https://www.bbc.com/zhongwen/trad/world-63940314

**Comment by bww0401** [comment]
我覺得科技始終始於人性，只要我們將他們用在正確的地方，應該不至於被取代，另外我覺得藝術相關領域更難被取代，因為對於美感的呈現，是人類的專長。

**Comment by 0212Jerry** [comment]
我認為追星的價值是在於我們認同明星的呈現出來的人格與人性，並且有所期待，例如我們會期待何時有演唱會等。儘管虛擬歌手可以給經紀公司帶來更大的利潤，也許很完美，但我個人認為這樣喜歡或崇拜著電腦生成出來的東西偏離了本質。也不免思考AI是否正逐漸影響我們信仰的標準，是否有一天我們所崇拜的偶像亦或是宗教都由AI取代，也許在基於成本考量下，虛擬寺廟、虛擬進香等產生也不是不可能。

**Comment by larry40305** [comment]
虛擬歌手最大的優點就是能突破身體限制，可以不用考慮體力或是舞蹈難度。

**Comment by boson13579** [comment]
但是大多數粉絲崇拜偶像是因為認為他們有能讓我們學習的良好品格，而AI創造出來的虛擬偶像卻缺少了一些人類情感上的溫度。因此，我認為AI虛擬偶像不能完全取代人類的偶像。

**Comment by yjpxi** [comment]
我覺得大部分人追星並不是一昧地跟隨流行，而是從偶像身上得到了精神安慰。然而由ai創造的虛擬偶像我覺得剛好可以減緩真人偶像超時工作的問題，因為虛擬偶像可以不用有生活作息的問題，但是透過新聞可以看到現在韓團偶像因為過度的工作，造成身體出現問題，所以我覺得虛擬偶像在這方面可以獲得很大的優勢。
